### PR TITLE
Ritchie wordlist

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
#659.

### CODE REVIEW

**Comment by chrzyki** [comment]
Is this okay like this @LinguList?

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -269,3 +269,4 @@ Kleinewillinghoefer-2014-16	Kleinewillinghoefer, Ulrich	2014		16	specific	Englis
...
+Ritchie-2019-110	Ritchie, Sandy and Sproat, Richard and Gorman, Kyle and van Esch, Dann and Schallhart, Christian and Bampounis, Nikos and Brard, Benoît and Fromseier Mortensen, Jonas and Holt, Millie and Mahon, Eoin	2019		110	questionnaire	English	Global	https://www.isca-speech.org/archive/Interspeech_2019/pdfs/2807.pdf	Ritchie2019		This list represents 110 numerals, used in the development of computational solutions to number verbalization in various languages. This initiative is part of a broader research effort at Google, enhancing the available machine learning techniques for building number names grammars across a wide range of languages for application in automatic speech recognition and text-to-speech synthesis systems.	3530-3534
```
Comment (xrotwang) [inline concepticondata/conceptlists.tsv]: It's **Daan** van Esch, not **Dann**.

File: concepticondata/references/references.bib [hunk 2]
```diff
@@ -2346,3 +2346,12 @@ @Misc{Nordlinger2019
   Url                                 = {https://arts.unimelb.edu.au/research-unit-for-indigenous-language/research/current-research-projects/50-words-project},
   Timestamp                           = {2019.10.23}
 }
+
+@Inproceedings{Ritchie2019,
+  Author                              = {Ritchie, Sandy and Sproat, Richard and Gorman, Kyle and van Esch, Dann and Schallhart, Christian and Bampounis, Nikos and Brard, Benoît and Fromseier Mortensen, Jonas and Holt, Millie and Mahon, Eoin},
```
Comment (xrotwang) [inline concepticondata/references/references.bib]: Same here, **Daan** not **Dann**

File: concepticondata/conceptlists.tsv (hunk 1 above)
Comment (chrzyki) [reply concepticondata/conceptlists.tsv]: Sorry I missed that, fixed.

File: concepticondata/references/references.bib (hunk 2 above)
Comment (chrzyki) [reply concepticondata/references/references.bib]: Fixed.

**Comment by chrzyki** [comment]
> See comments. Also, when adding a concept list, you should run
> 
>     * `concepticon stats`
> 
>     * `concepticon make_linkdata` and
> 
>     * `concepticon make_app`
> 
> 
> to recreate the derived data.

Done!

**Comment by chrzyki** [comment]
@xrotwang Okay like this?

**Pull request merged**